examples/abbas_work_area/file_integrity.py

```diff
@@ -0,0 +1,46 @@
+import hashlib
+
+
+def generate_sha256(file_path):
+    sha256 = hashlib.sha256()
+
+    with open(file_path, 'rb') as input_file:
+        for blocks in iter(lambda: input_file.read(4096), b""):
```

Comment:
Please provide comments so a dev understands waht b"" is doing as a second argument to iter(), also if we are reading the file as "rb", do we need it ?